docs/standard/index.yml

```diff
@@ -18,6 +18,8 @@ landingContent:
     linkLists:
       - linkListType: overview
         links:
+          - text: Download the .NET SDK
```

Comment:
We've put the download link in a "download" linklisttype in other landing pages. We should be consistent.